src/parser/decompiler/decompile.sh

```diff
@@ -1,32 +1,35 @@
-
+!#/bin/sh
+if [ -f "../../../.env" ]; then
 . ../../../.env #Retrieve config paths
+fi
+
+# Windows vs. linux package
+if [ -f "$DECOMPILER_PATH/Decompiler.exe"  ]; then
+  DEC_CMD="$DECOMPILER_PATH/Decompiler.exe"
+  else
+  DEC_CMD="$DECOMPILER_PATH/Decompiler"
+fi
```

Comment:
Would it be easier to have `DECOMPILER_PATH` also include the file name?

Reply:
Yes, much easier!  Fixed